From a9d198b6e1d8b97049ece43d857fa80ae4c0e5ea Mon Sep 17 00:00:00 2001
From: Harry Wentland <harry.wentland@amd.com>
Date: Thu, 13 Apr 2017 10:29:28 -0400
Subject: [PATCH 1218/2094] drm/dp-helper: DP_TEST_MISC1 should be
 DP_TEST_MISC0

Bring this in line with spec and what commit in upstream drm tree.

v2: Update DC usage of DP_TEST_MISC0

Signed-off-by: Harry Wentland <harry.wentland@amd.com>
Reviewed-by: Alex Deucher <alexander.deucher@amd.com>
---
 drivers/gpu/drm/amd/display/dc/core/dc_link_dp.c | 2 +-
 include/drm/drm_dp_helper.h                      | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/gpu/drm/amd/display/dc/core/dc_link_dp.c b/drivers/gpu/drm/amd/display/dc/core/dc_link_dp.c
index 802d8cc..9f12ba8 100644
--- a/drivers/gpu/drm/amd/display/dc/core/dc_link_dp.c
+++ b/drivers/gpu/drm/amd/display/dc/core/dc_link_dp.c
@@ -1691,7 +1691,7 @@ static void dp_test_send_link_test_pattern(struct core_link *link)
 			sizeof(dpcd_test_pattern));
 	core_link_read_dpcd(
 			link,
-			DP_TEST_MISC1,
+			DP_TEST_MISC0,
 			&dpcd_test_params.raw,
 			sizeof(dpcd_test_params));
 
diff --git a/include/drm/drm_dp_helper.h b/include/drm/drm_dp_helper.h
index 4b14a76..d6a5015 100644
--- a/include/drm/drm_dp_helper.h
+++ b/include/drm/drm_dp_helper.h
@@ -419,7 +419,7 @@
 
 #define DP_TEST_PATTERN			    0x221
 
-#define DP_TEST_MISC1                       0x232
+#define DP_TEST_MISC0                       0x232
 
 #define DP_TEST_CRC_R_CR		    0x240
 #define DP_TEST_CRC_G_Y			    0x242
-- 
2.7.4

